images/scripts/rhel.setup

```diff
@@ -220,6 +221,11 @@ if [ "$IMAGE" = "centos-8-stream" ]; then
     TEST_PACKAGES="${TEST_PACKAGES/kpatch-dnf /}"
 fi
 
+# wireguard-tools not available on rhel-8* and centos-8-stream
+if [ "$IMAGE" = "rhel-8*" ] || [ "$IMAGE" = "centos-8-stream" ]; then
```

Comment:
This "globbing" doesn't work. The way to write that in shell is `[ "${IMAGE#rhel-8*}" != "$IMAGE" ]`, see e.g. line 53.

Reply:
Fixed it, thanks!